gnomad/utils/vcf.py

```diff
@@ -684,6 +758,7 @@ def make_info_dict(
         - INFO fields for filtering allele frequency (faf) annotations
 
     :param prefix: Prefix string for data, e.g. "gnomAD". Default is empty string.
+    :param suffix: Suffix string for data, e.g. "gnomAD". Default is empty string.
```

Comment:
suffix is usually gnomAD too? Or do you have another example?

Reply:
It can be "gnomad" or "non_ukb", would you prefer non_ukb here?

Reply:
ah, you mean we can put "gnomad" and "non_ukb" either as a prefix or a suffix, if that's the case, it's fine.